### PR TITLE
Version bump to 1.7.0 for Release

### PR DESCRIPTION
**Pull request by aayush3011**
## Type of PR

- [ ] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Observability + Testing
- What changes or considerations did you make in relation to observability?

- Did you add testing to account for any new or changed work?

## Issues Closed or Referenced

- Closes #<issue number> (this will automatically close the issue when the PR closes)
- References #<issue number> (this references the issue but does not close with PR)

### CODE REVIEW

**Review by FabianMeiswinkel — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Release History
 
+### 1.7.0
+#### New Features
+* Updated `azure-cosmos` version to 4.41.0.
```
Comment (xinlian12) [inline CHANGELOG.md:5]: should we upgrade to 4.42.0? it released this morning

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 ## Release History
 
+### 1.7.0
```
Comment (kushagraThapar) [inline CHANGELOG.md:3]: Missing date here

File: doc/README_Sink.md [hunk 3]
```diff
@@ -168,19 +168,23 @@ To delete the created Azure Cosmos DB service and its resource group using Azure
...
+| connect.cosmos.sink.bulk.enabled | boolean | Flag to indicate whether bulk mode is enabled. By default it is `true`.                                                                                      | Optional |
+| connect.cosmos.sink.maxRetryCount | int     | Max retry attempts on transient write failures. By default it is `10` times. NOTE: This is different from max throttling retry attempts, which are infinite. | Optional |
+| connect.cosmos.connection.sharing.enabled | boolean | Flag to enable connection sharing between instances of cosmos clients on the same jvm. NOTE: If you have set 'connect.cosmos.connection.gateway.enabled' to true, then this configure will not make any difference.                                                                | Optional |
+| key.converter | string  | Serialization format for the key data written into Kafka topic                                                                                               | Required |
+| value.converter | string  | Serialization format for the value data written into the Kafka topic                                                                                         | Required |
```
Comment (kushagraThapar) [inline doc/README_Sink.md:184]: @xinlian12 this looks like a new feature / fix. I don't see it in the changelog. Can you please check?

**Review by kushagraThapar — CHANGES_REQUESTED** [review]

File: doc/README_Sink.md (hunk 3 above)
Comment (kushagraThapar) [reply doc/README_Sink.md:184]: nvm, ignore my above comment.

**Review by kushagraThapar — APPROVED** [review]
@aayush3011 please add the date to release changelog. otherwise looks good to me, thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (xinlian12) [reply CHANGELOG.md:5]: Discussed offline, keep 4.41.0

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**